protopipe/aux/example_config_files/protopipe/performance.yaml

```diff
@@ -1,6 +1,8 @@
+
 general:
  # Directory with input data file
  # [...] = your analysis local full path OUTSIDE the Vagrant box
+ # Please refer to directory structure shown at Lugano
```

Comment:
please remove this, because the directory structure that comes with protopipe-grid-interface >=0.2.0 has been updated from the Lugano version